web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -149,10 +149,20 @@ export class ShardComponent implements OnInit, OnDestroy {
   }
 
   getUrl(tablet) {
-    return `http://${tablet.hostname}:${tablet.port_map.vt}`;
+    return `https://${tablet.hostname}--${this.squareApp()}.${this.squareSuffix()}`;
```

Comment:
i'm guessing you don't want that upstreamed

Reply:
eep!